File: PB - More Exercises JavaScript/2.0 Conditional Statements - More Exercises/02. Sleepy Tom Cat.js
function sleepyTomCat(days) {
    let gameTomMin = 30000
    let year = (365 - days)
    let yearToMin = year * 63 + days * 127
    let subtotal = Math.abs(gameTomMin - yearToMin)
    let hours = Math.trunc(subtotal / 60)
    let min = subtotal % 60
  
    if (gameTomMin >= yearToMin) 
        return `Tom sleeps well\n${hours} hours and ${min} minutes less for play`
    return `Tom will run away\n${hours} hours and ${min} minutes more for play`
  }

//   console.log(sleepyTomCat(20))

File: PB - More Exercises GO/2.0 Conditional Statements - More Exercises/02. Sleepy Tom Cat.go
package main

import (
	"fmt"
	"math"
)

func main() {
	var days float64
	fmt.Scan(&days)
	gameTomMin := 30000.00
	year := (365 - days)
	yearToMin := year*63 + days*127
	subtotal := math.Abs(gameTomMin - yearToMin)
	hours := math.Trunc(subtotal / 60)
	min := math.Mod(subtotal, 60)

	if gameTomMin >= yearToMin {
		fmt.Printf("Tom sleeps well\n%v hours and %v minutes less for play", hours, min)
	} else {
		fmt.Printf("Tom will run away\n%v hours and %v minutes more for play", hours, min)
	}

}

/*
name   :02. Sleepy Tom Cat
input  :20
output :
Tom sleeps well
95 hours and 25 minutes less for play
*/
